services/QuillLMS/app/models/question.rb

```diff
@@ -165,7 +165,38 @@ def connect_sentence_combining?
   end
 
   # Translatable
-  def self.translatable_field_name = 'instructions'
+  def self.default_field_name = 'instructions'
+
+  def translatable_data(type:)
+    if stored_as_array?(type)
+      translatable_for_array(type)
+    else
+      translatable_for_hash(type)
+    end
+  end
+
+  def create_translation_mappings
+    super
+    create_data_translation_mappings(type: INCORRECT_SEQUENCES)
+    create_data_translation_mappings(type: FOCUS_POINTS)
+  end
+  def create_data_translation_mappings(type:)
+    translatable_data(type:).each do |field_name, translatable_text|
+      create_translation_mappings_with_text(field_name:, translatable_text:)
+    end
+  end
+
+  private def translatable_for_hash(type)
+    data[type]&.transform_values { |v| v['feedback'] }
+      &.compact
+      &.transform_keys { |k| "incorrectSequences.#{k}" } || {}
+  end
+
+  private def translatable_for_array(type)
+    data[type].each_with_object({}) do |item, hash|
+      hash["incorrectSequences.#{item['uid']}"] = item['feedback']
+    end
+  end
```

Comment:
Take or leave this, but I wonder if there's an easier-to-read approach here where there's a method to force the shape of the array version of the data to be the same as the hash version of the data, and then a second method that does the translation-focused transform.  That way if you tweak the logic of the transform you don't have to do it in two places.

Also, I note that this looks like it hard-codes `incorrectSequences` into the result key names.  Does this work with Focus Points?